### PR TITLE
Buttons and Spinner

### PR DESCRIPTION
**Pull request by mantagen**

### CODE REVIEW

File: reg-simple-ui/components/Button/ButtonCircle.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { Welcome } from '@storybook/react/demo';
```
Comment (Jbarget) [inline reg-simple-ui/components/Button/ButtonCircle.stories.js]: dont need this for all these files

File: reg-simple-ui/components/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      top: 0px;
+      right: 0px;
+      bottom: 0px;
+      left: 0px;
+      background: rgba(255, 255, 255, 0.08);
+    }
+
+    &:active::after {
+      content: '';
+      position: absolute;
```
Comment (Jbarget) [inline reg-simple-ui/components/Button/Button.js]: i dont know why these are absolute, can you explain me?

File: .storybook/config.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import { configure } from '@storybook/react';
 
+import '../reg-simple-ui/util.css';
```
Comment (Jbarget) [inline .storybook/config.js]: when is this used in storybook?

File: reg-simple-ui/util.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
+/*
+
+  From: https://gist.github.com/ffoodd/000b59f431e3e64e4ce1a24d5bb36034
+*/
+.sr-only {
+  border: 0 !important;
+  clip: rect(1px, 1px, 1px, 1px) !important; /* 1 */
```
[the remark this answers is not in the document]
Comment (mantagen) [reply reg-simple-ui/util.css]: removed

File: reg-simple-ui/components/Button/ButtonCircle.stories.js (hunk 1 above)
Comment (mantagen) [reply reg-simple-ui/components/Button/ButtonCircle.stories.js]: removed

File: .storybook/config.js (hunk 3 above)
Comment (mantagen) [reply .storybook/config.js]: for `.sr-only` in the buttons

**Pull request merged**